insights/parsers/openstack_server_list_all.py

```diff
@@ -0,0 +1,98 @@
+"""
+openstack server list --all command - Command
+=============================================
+"""
+
+from insights import CommandParser, parser, LegacyItemAccess
+from insights.parsers import ParseException, SkipException, get_active_lines, parse_delimited_table
+from insights.specs import Specs
+
+
+@parser(Specs.openstack_server_list_all)
+class OpenstackServerListAll(LegacyItemAccess, CommandParser):
```

Comment:
Your use of `LegacyItemAccess` here is not really appropriate.  It really should not be used for new parsers, and it is meant to be used with a `self.data` that is a of type `dict` instead of `list` as you have in this parser.

You could simplify this parser and make it easier to understand and use if you stored the output in a `dict` based on `Name` which appears to be the primary access mechanism to this data.  So for instance instead of creating `self.data` which is a list, you could create `self.servers` which is a `dict` indexed by the key in the `Name` column, and containing the existing line dict as the value.  You only need to keep a list if order of the items matters.  

The second simplification would be to remove `LegacyItemAccess` and use a `dict` mixin class to your parser as was implemented in the [Partitions parser](https://github.com/RedHatInsights/insights-core/blob/master/insights/parsers/partitions.py).  This would allow you to access the `self.servers` dictionary directly from the parser object, like `parser_result['compute-0']`, and you could provide methods to access the other information that would be useful.